client/components/password-change/password-change.js

```diff
@@ -45,12 +49,16 @@ export default class PasswordChange extends React.Component {
         newPassword1,
         newPassword2,
       }),
-    }).then(() => {
+    }).then((response) => {
+      setTimeout(() => toast.success(response.data.detail), 0);
```

Comment:
Why are you using setTimeout and not calling it directly?

Reply:
@Vivekrajput20 I am using setTimeout 0 because it calls toast after some amount of time (like 1 tick), and during this time our url will be changed (check next line), so our toast will be displayed on the new page.